index.md

```diff
@@ -1,3 +1,6 @@
+---
+editLink: false
```

Comment:
I would keep this one, our guide list is manually curated for now.

Reply:
Are you sure? I find it weird to arrive on a website which offers me to edit the homepage.

Reply:
Let's be open by default ;-)